test/github-issues/9600/issue-9600.ts

```diff
@@ -0,0 +1,60 @@
+import "reflect-metadata"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { DataSource } from "../../../src"
+import { FooEntity } from "./entity/Foo"
+import { expect } from "chai"
+
+// if bug persists, will throw error when creating testing connection, won't get to the test
+
+describe("github issues > #9600 if set closure-table primary key unsigned, it occurs FK error", () => {
```

Comment:
We're moving towards placing tests in the [functional tests directory](https://github.com/typeorm/typeorm/tree/master/test/functional) since it's easier to see what is covered and not duplicate tests.

We already have `test/functional/tree-tables/closure-table/closure-table.ts`where these tests could go.